lessons/sentiment-analysis.md

```diff
@@ -397,6 +397,12 @@ But even at this level, *VADER* also runs into a number of errors. The sentence
 
 What do you notice about the distribution of scores? How can you imagine collecting them in a manner that would help you better understand your data and its relationships to the research questions you care about? (Feel free to experiment with different kinds of text in the *message_text* variable to see how the tool responds to different types of language constructions). The code you have just written can be repurposed for any text.
 
+# Acknowledgments
+
+My sincere thanks to Justin Joque, Visualization Librarian at the University of Michigan Library and the Digital Project Studio for support in formulating the ideas and approach behind this lesson. More information about the Digital Project Studio can be found at https://clarkdatalabs.github.io
```

Comment:
Can you make https://clarkdatalabs.github.io either a clickable link (wrap in `<` and `>`) or maybe even more simply just make the first mention of "Digital Project Studio" a link to the site?